internal/checker/utilities.go

```diff
@@ -2073,3 +2084,211 @@ func (c *Checker) checkNotCanceled() {
 		panic("Checker was previously cancelled")
 	}
 }
+
```

Comment:
These are some functions I added temporarily to make Signature help work. They will be removed once the node builder pr is merged. Also, these do not give the best result for generics, and returns `object` instead. This is why some of the tests have a comments `//returns object` because this part returns `object`.